### DISCUSSION TITLE
About suggesting weapons

### DISCUSSION THREAD

**Discussion by ImBoredSoHereIAm** [Hydraxous/UltraFunGuns#13] [Q&A]
Are you going to be accepting weapon/weapon variant ideas from others, or do you mainly just want to focus on your own ideas? I have a lot of stuff I've written that's sitting in the backburner. Most of them are stuff I wrote and posted to the sub, like a rework concept for the screwdriver or a weapon variant for the shotgun. I just wondered if I could suggest those ideas to someone more talented, since I am basically inept at coding.

Pardon me because I only recently made a Github account and might not understand some common etiquette.

**Comment by Hydraxous** [comment]
As a whole my priority for now is creating unique weapons rather than variants. That being said, I am mainly focusing on weapons of my own design. However, that doesn't mean I wont add concepts which resonate really well with what I'm trying to make as-well as the game. You are welcome to post your suggestions, ideas, concepts, etc. [here](https://github.com/Hydraxous/UltraFunGuns/discussions/categories/suggestions). 

Since it's just I alone creating these weapons, it takes a considerable amount of time and effort so I have to very carefully choose where I spend it.

**Reply by ImBoredSoHereIAm** [reply]
Thank you for answering. I'll still post my variant ideas in case you want to pick them up (or someone else sees them), and I'll also adjust them to work with your weapon ideas (keep in mind that I am copy and pasting from the sub). You're making really awesome stuff btw. Even with the keybind limitations, going between the baller and my marksman is just too fun and something I regularly do now.
